errata_tool/erratum.py

```diff
@@ -1093,7 +1124,7 @@ def __str__(self):
             "\n  batch_id:    " + str(self.batch_id) + \
             "\n  ship date:   " + str(self.ship_date) + \
             "\n  age:         " + str(self.age) + " days" \
-            "\n  bugs:        " + str(self.errata_bugs) + \
+            "\n  bugs:        " + str(self.errata_bugs + self.jira_issues) + \
```

Comment:
Perhaps the two kinds should be separate on the output if we take the time to keep then separate in the data structure

Reply:
Good point. Updated :+1: